app/api/export/index.test.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import '../ada/lib/test-config'; // TODO: needs to import this from ADA API, should be somewhere common at TOP level
```

Comment:
what is the plan for the TODOs in the PR? are being tracked somehow?

Reply:
was not tracked outside.
Now added : https://github.com/Emurgo/yoroi-frontend/issues/250